programs/stake/src/stake_state.rs

```diff
@@ -4721,7 +4725,7 @@ mod tests {
         let stake_pubkey = solana_sdk::pubkey::new_rand();
         let rent = Rent::default();
         let rent_exempt_reserve = rent.minimum_balance(std::mem::size_of::<StakeState>());
-        let stake_lamports = rent_exempt_reserve + MINIMUM_STAKE_DELEGATION;
+        let stake_lamports = rent_exempt_reserve + MINIMUM_STAKE_DELEGATION * 2;
```

Comment:
I may be missing something, but this test always splits `stake_lamports`, so I think it should stay as it is

Reply:
I agree. Done!